### ISSUE TITLE
Improve button contrast

### ISSUE THREAD

**Issue by zenorocha** [dracula/flowlab#1]
When using purple as the background, it's recommended to use black instead of white for the text color. This will improve the text legibility and accessibility.

![CleanShot 2022-02-26 at 07 15 53@2x](https://user-images.githubusercontent.com/398893/155848427-87a7e4c1-da77-4dc3-8421-fb4936b7e2ca.png)
